### PR TITLE
Chore/update apply form

### PR DESCRIPTION
**Pull request by C-Ivan-Bandilla**
### 🛠 Changes
- I added the present address field input with its corresponding validation.
- Added the permanent field with some validation
- add civil status field
- Added an email acknowledgment event in submitting application
- Also fix the ./script not found on vite error

### ✨ Context
What's the context for the changes? Are there any?

### 🧠 Rationale
I adapted the code from @Carl-Tabuso in create account form but since livewire wizard has [issue ](https://github.com/spatie/laravel-livewire-wizard/discussions/100) I refrained from using form objects and instead use FormRequest 
- I tried [this](https://forum.laravel-livewire.com/t/programmatically-setting-input-value-doesnt-trigger-wire-model-update/862/3)  too  and [this](https://github.com/livewire/livewire/discussions/7905) just documenting
- I haven't tried [this yet](https://livewire.laravel.com/docs/forms#custom-form-controls:~:text=HTML%20is%20the-,x%2Dmodelable%3D%22count%22,-and%20%7B%7B%20%24attributes%20%7D%7D)

### 📸 Screenshots (Optional)
image

### 🎥 Demos (Optional)
https://drive.google.com/file/d/1F-TPdL1oqqRt3xxiBAeF_S1pl-Gpv5uV/view?usp=sharing

### ✔️ Checklist
This is a set of criteria to ensure the correctness of your PR. Check each that applied.
- [x] Is `staging` the base branch of this PR?
- [x] Do your changes not reveal sensitive information, such as secrets, API keys, etc?
- [ ] Are there no erroneous console logs, debuggers, or leftover code in your changes?

### CODE REVIEW

File: app/Http/Controllers/Application/ApplicantController.php [hunk 1]
```diff
@@ -134,13 +138,29 @@ public function store(Request|array $request, bool $isValidated = false)
...
+            UserPermission::VIEW_ALL_PENDING_APPLICATIONS->value,
+            UserPermission::VIEW_ALL_QUALIFIED_APPLICATIONS->value,
+            UserPermission::VIEW_ALL_PRE_EMPLOYED_APPLICATIONS->value,
+        ];
+
+        $usersWithPermissions = User::permission($permissions)->get();
+
+        foreach ($usersWithPermissions as $otherUser) {
+            $otherUser->notify(new AccountCreated($user->user_id, $application->application_id, ['database', 'broadcast']));
+        }
```
Comment (C-Ivan-Bandilla) [inline app/Http/Controllers/Application/ApplicantController.php:155]: I add notification to the other user

The structure of the notification is not yet final
        return [
            'type' => 'new-applicant',
            'label' => ['applicants', 'new'],
            'userId' => $this->userId,
            'model' => 'applications',
            'modelId' => $this->applicationId,
        ];

**Comment by Carl-Tabuso** [comment]
Will you need new models and migrations for migrating user data from `applicant_experiences` `applicant_education` and `applicant_skills` to let's say `employee_experiences` `employee_education` and `employee_skills`?

**Comment by C-Ivan-Bandilla** [comment]
I think yes

**Comment by Carl-Tabuso** [comment]
Will open a pr for this later.

**Review by Carl-Tabuso — APPROVED** [review]

**Pull request merged**